autodistill/helpers.py

```diff
@@ -84,3 +89,41 @@ def split_data(base_dir, split_ratio=0.8):
             "names": names,
         }
         yaml.dump(data, file)
+
+
+def visualize_predictions(
+    file_name: str, detections: sv.Detections, annotation_type: str
+) -> None:
+    if annotation_type not in VALID_ANNOTATION_TYPES:
+        raise ValueError(
+            f"Invalid annotation type. Must be one of {VALID_ANNOTATION_TYPES}"
+        )
+
+    if annotation_type == "box":
+        annotator = sv.BoxAnnotator()
+    elif annotation_type == "mask":
+        annotator = sv.MaskAnnotator()
+
+    image = cv2.imread(file_name)
+
+    image_bgr = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
+
+    annotated_image = annotator.annotate(scene=image_bgr, detections=detections)
+
+    sv.plot_image(annotated_image)
+
+
+def split_video_frames(video_path: str, output_dir: str, stride: int) -> None:
+    video_paths = sv.list_files_with_extensions(
+        directory=video_path, extensions=["mov", "mp4"]
```

Comment:
Is this case insensitive? If not, many videos (eg from iPhones) won't get detected because they're capitalized (eg `.MOV`) by default. Here or in Supervision we should probably account for that.

Reply:
@yeldarby our `list_files_with_extensions` is case-sensitive.